CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # UNRELEASED
 
+-   Report failures to callback when importing backed-up room keys. The
+    callback is now called with a third argument, giving the number of invalid
+    room keys.
```

Comment:
It would be helpful to mention which API is affected here. `OlmMachine.importBackedUpRoomKeys` I think?